docs/CONTRIBUTING.md

```diff
@@ -505,3 +522,16 @@ You can view these changes locally on the `localhost:1314` port.
 Note that changes made to the docs in the sandbox
 will be overwritten so the same changes should be applied
 to the corresponding doc source in the `lifecycle-toolkit` documentation.
+
+## Tips and Tricks (aka How to Avoid Gotchas)
+
+1. Sign every commit! `git commit -sm "message"`
+1. Always run `cd docs` then `make markdownlint` to show issues.
+1. If your PR has one commit, make it semantic.
+Make it all lowercase.
+Make the commit message conform to the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) standard:
+
+`git commit -sm "docs: i did something"`
+
+1. PRs with a single commit are very strictly checked for the above conventional commit messages.
```

Comment:
I think this is not required anymore. Single commit PRs do accept any signed commit message -> PR title is the main source of thruth

Reply:
exactly! single commit PRs are handled the same way as any other PRs now. this restriction was removed now since github finally removed their weird behaviour around single commit PRs in combination with squash and merging

Reply:
I made suggestions to modify the text.